src/lib.rs

```diff
@@ -209,8 +252,8 @@ fn decrypt<'p>(
 ) -> PyResult<Bound<'p, PyBytes>> {
     let identities = identities.iter().map(|pi| pi.as_ref().as_identity());
 
-    let decryptor =
-        age::Decryptor::new(ciphertext).map_err(|e| DecryptError::new_err(e.to_string()))?;
+    let decryptor = age::Decryptor::new(ArmoredReader::new(ciphertext))
+        .map_err(|e| DecryptError::new_err(e.to_string()))?;
```

Comment:
Just confirming my understanding: this also works for non-armored inputs by falling back to a raw read internally, right? That's what I'm seeing from `ArmoredReader`'s implementation but I want to make sure that matches your understanding too.

Reply:
Yep, I believe that's how it works. Since the binary form begins with `age-encryption...` and the armoured begins with `\s*-----...` (regex style), it can probably does a best effort detection to see if it is a binary and otherwise will try parsing as armoured.